client.py

```diff
@@ -1,252 +1,144 @@
+"""
```

Comment:
Is the license required? We were not using it before

Reply:
Yes, Avtar asked me to add it.